org-code-javabuilder/theater/src/main/java/org/code/theater/Theater.java

```diff
@@ -0,0 +1,196 @@
+    */
+   public static void wait(double seconds) {
+   }
+
+   /**
+    * Clear everything, starting from the beginning and emptying the canvas.
+    */
+   public static void reset() {
+   }
+
+   /**
+    * Draw an image on the canvas at the given location, expanded or shunk to fit
+    * the width and height provided
+    * 
+    * @param file     the name of the file in the asset manager
+    * @param x        the left side of the image in the canvas
+    * @param y        the top of the image in the canvas
+    * @param width    the width to draw the image on the canvas
+    * @param height   the height to draw the image on the canvas
+    * @param rotation the amount to rotate the image
+    * @throws FileNotFoundException if the file can't be found in the project.
+    */
+   public static void drawImage(String filename, double x, double y, double width, double height, double rotation)
+         throws FileNotFoundException {
+   }
+
+   /**
+    * Draw an image on the canvas at the given location, expanded or shunk to fit
+    * the width and height provided
+    * 
+    * @param img      the Image object to draw on the canvas
+    * @param x        the left side of the image in the canvas
+    * @param y        the top of the image in the canvas
+    * @param width    the width to draw the image on the canvas
+    * @param height   the height to draw the image on the canvas
+    * @param rotation the amount to rotate the image
+    */
+   public static void drawImage(Image img, double x, double y, double width, double height, double rotation) {
+   }
+
+   /**
+    * Draws text on the image.
+    * 
+    * @param text   the text to draw
+    * @param x      the distance from the left side of the image to draw the text.
+    * @param y      the distance from the top of the image to draw the text.
+    * @param color  the color to draw the text, using any CSS color string (e.g. #234 or green)
+    * @param font   the name of the font to draw the text in
+    * @param height the height of the text in pixels.
+    */
+   public static void drawText(String text, double x, double y, String color, String font, double height) {
+   }
+
+   /**
+    * Draw a line on the cavas.
+    * 
+    * @param startX the beginning X coordinate of the line.
+    * @param startY the beginning Y coordinate of the line.
+    * @param endX   the end X coordinate of the line.
+    * @param endY   the end Y coordinate of the line.
+    */
+   public static void drawLine(double startX, double startY, double endX, double endY) {
+   }
+
+   /**
+    * Draw a regular polygon on the canvas.
+    * 
+    * @param x     the center X coordinate of the polygon
+    * @param y     the center Y coordinate of the polygon
+    * @param sides the number of sides of the polygon
+    * @param size  the distance from the center to each point on the polyon
+    */
+   public static void drawPolygon(double x, double y, int sides, double size) {
```

Comment:
NIT: Should this be `drawRegularPolygon`?

Reply:
👋 Hi. Former high school geometry teacher here. Sorry for eavesdropping, but I might also suggest...
- +1 to Jessica's comment that this only works for a regular polygon
- I think x and y are the coordinates of the _center_ of the polygon. Phrasing it as "of the polygon" is ambiguous - is it one of the vertices? The center? etc
- I'd suggest using the term "radius" instead of "size". The definition - "distance from the center to each point" - matches the definition of the radius of a regular polygon, which is how it's taught in high school geometry - https://www.mathopenref.com/polygonradius.html.

Reply:
Yes, I'll update it.